src/tool/natvis/packages.version.props

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- NOTE: This needs to stay in sync with packages.config -->
```

Comment:
Should we be generating this file + packages.config with cmake to ensure these are in sync?

Reply:
in due course - patterned this after Gregg Miskelly's VS SDK samples

Reply:
However, I'd likely generate this from the vcxproj, to maintain standalone behavior.  I'd like to keep the cmakelists.txt as a pure msbuild invoker.

Reply:
Makes sense. File an issue to track please

Reply:
turns out the props file wasn't even needed